src/app/onboard/member/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import React, { useState } from "react";
+import { useRouter } from "next/navigation";
+import { IoArrowBackOutline } from "react-icons/io5";
+
+import { Button } from "@/components/Button";
+import { MembershipTypeButton } from "@/components/MembershipTypeButton";
+
+const MembershipType = () => {
+  const [selectedMembership, setSelectedMembership] = useState<
+    boolean | undefined
+  >(undefined);
+
+  const handleButtonClick = (isMember: boolean) => {
+    setSelectedMembership(isMember);
+  };
+
+  const router = useRouter();
+
+  const handleBackButtonClick = () => {
+    router.back();
+  };
+
+  const handleNextButtonClick = () => {
+    // Navigate to the home page
+    router.push("/");
+  };
+
+  return (
+    <div className="flex flex-col h-dvh mx-4 gap-y-6">
+      <div className="flex mt-4">
+        <button
+          className="flex items-center px-4 hover:text-primary"
+          onClick={handleBackButtonClick}
+        >
+          <IoArrowBackOutline />
+        </button>
+        <div className="text-lg font-medium">Select your membership type</div>
+      </div>
+
+      <MembershipTypeButton
+        selectedMembership={selectedMembership === true}
```

Comment:
you can do this as selectedMembership is a boolean
```suggestion
        selectedMembership={selectedMembership}
```